src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +50,36 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService) injector
+                .getInstance(AuthenticationService.class);
+        try {
+            authenticationService.register("bob@mail.com", "password");
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Can not register user", e);
```

Comment:
The message in the `RuntimeException` should reflect the registration operation, not login. Consider changing it to something like 'Registration unsuccessful'.